sql/sql_prepare.cc

```diff
@@ -3958,7 +3958,6 @@ Prepared_statement::execute_loop(String *expanded_query,
   return error;
 }
 
-
```

Comment:
According to our coding style there should be 2 lines between functions. This line should not have been removed.